src/types/task.list.ts

```diff
@@ -0,0 +1,52 @@
+import { DateTime } from "luxon";
+import { readableFormat } from "../utils/date.formatter";
+
+export interface TaskList {
+    completedTasks: string;
+    email: string;
+    exemption: string;
+    activeOfficers: string;
+    activePscs: string;
+    activeMembers: string;
+    additionalPscs: string;
+    additionalOfficers: string;
+    additionalMembers: string;
+    members: string;
+    moment: string;
```

Comment:
moment looks like the prototype was using the moment library to generate a date. seeing as we aren't using moment, rename it to something better like 'todaysDate'